web/src/lib/components/map/mapUtils.ts

```diff
@@ -39,7 +39,7 @@ import type {
  */
 const cssVars = {
```

Comment:
I think this can be removed and its usages can be directly replaced with `getCssVariable('...')`